app/src/main/java/it/niedermann/owncloud/notes/android/activity/NoteActivity.java

```diff
@@ -37,14 +36,27 @@ protected void onCreate(Bundle savedInstanceState) {
         if (savedInstanceState != null) {
             note = (DBNote) savedInstanceState.getSerializable(PARAM_NOTE);
         }
-        actionBar = getSupportActionBar();
+        ActionBar actionBar = getSupportActionBar();
         if (actionBar != null) {
             actionBar.setTitle(note.getTitle());
             actionBar.setSubtitle(DateUtils.getRelativeDateTimeString(getApplicationContext(), note.getModified().getTimeInMillis(), DateUtils.MINUTE_IN_MILLIS, DateUtils.WEEK_IN_MILLIS, 0));
         }
         noteContent = (RxMDTextView) findViewById(R.id.single_note_content);
 
-        RxMarkdown.with(note.getContent(), this)
+        String content = note.getContent();
+        // The following replaceAll adds links ()[] to all URLs that are not in an existing link.
+        // This regular expression consists of three parts:
+        // 1. (?<![(])
+        //    negative look-behind: no opening bracket "(" directly before the URL
+        //    This prevents replacement in target part of Markdown link: [](URL)
+        // 2. (https?://[-a-zA-Z0-9+&@#/%?=~_|!:,.;]*[-a-zA-Z0-9+&@#/%=~_|])
+        //    URL pattern: matches all addresses beginning with http:// or https://
+        // 3. (?![^\\[]*\\])
+        //    negative look-ahead: no closing bracket "]" after the URL (otherwise there have to be an opening bracket "[" before)
+        //    This prevents replacement in label part of Markdown link: [...URL...]()
```

Comment:
Can you please use the `/* */`-syntax on multi line comments? I'm not a pro in regex, but i think it should do the job 👍 (I'll ask a friend later who is better in regex)